src/static/css/module/helpers.less

```diff
@@ -253,11 +253,47 @@ object {
   margin-left: -1.2em;
 }
 
-.alert:before {
-  .cf-icon;
-  content: "\e103";
-  color:  @redorange;
-  text-indent: -1.4em;
+.checked {
+  position: relative;
+
+  &:before,
+  li&:before {
+    .cf-icon();
+    content: "\e100";
+    font-size: 17px;
+    color: @green;
+    left: -21px;
+  }
+  li&.list_item__spaced:before {
+    top: 3px;
+  }
+  p&:before {
+    text-indent: -1.4em;
+  }
+  li& {
+    list-style: none;
+  }
+}
+
+.alert {
+  &:before,
+  li&:before {
+    .cf-icon;
+    content: "\e103";
+    color:  @redorange;
+    font-size: 17px;
+    left: -21px;
+  }
+  li&.list_item__spaced:before {
+    top: 3px;
+  }
+  p&:before {
+    text-indent: -1.4em;
+  }
+  li& {
+    position: relative;
+    list-style: none;
+  }
 }
 
```

Comment:
I'm wondering if we could create a class (like `.icon-bullet`) that would pull out shared code here (there's also a `.custom-bullet` macro in `.cf-typography`, but that wouldn't really reduce the code.) 

I also notice that I hard-coded the font sizes here; I will look at using relative sizes instead. ;)